src/components/battleGroundDetails.tsx

```diff
@@ -47,482 +38,97 @@ const BattleGroundDetails = () => {
     // const [attIdxArray, setAttIdxArray] = useState<number[]>([]);
     const [checkedPosition, setCheckedPosition] = useState(false);
 
-    const handleChangeCheckbox = () => {
+    const handleGameVersionChange = (event: SelectChangeEvent) => {
+        const version = event.target.value;
+
+        // TODO improve method of switching between versions.
```

Comment:
One way to tackle this would be to create a map of versions like:
```js
const versionConfigs = {
108: v108Config,
109: v109Config
...etc
}
```

Then just load the config with `setVersionConfig(versionConfigs[version])`